scripts/umbrel-os/external-storage/monitor

```diff
@@ -38,6 +44,14 @@ main () {
       break
     fi
 
+    if [[ $(dmesg | grep --quiet "uas_abort_handler") > 0 ]]; then
+      echo "External storage might have UAS issues"
+      sed "s/usb-storage.quirks=/usb-storage.quirks=$(get_uas_string),/g" -i /boot/cmdline.txt
```

Comment:
Is there ever a situation where `usb-storage.quirks=` could be empty so we end with a comma and break it?

Reply:
No, except the user emptied it, but then it's the users fault. This has been in Umbrel OS since v0.1.1, an empty list was never there, when the list doesn't exist at all, sed would do nothing. An empty list would maybe even cause the kernel to refuse boot because there is nothing after the =, but I'm not sure about that